### DISCUSSION TITLE
Sandbox API docs "Hello World" quick start compilation error

### DISCUSSION THREAD

**Discussion by IwateKyle** [hecrj/iced#1512] [Q&A] [answered]
I've copied the quick start "Hello World from 

https://docs.iced.rs/iced/trait.Sandbox.html#a-simple-hello-world

but I get the following compilation error regarding the view method.  It seems that the functions 
".....".into()
doesn't compile.

error snippet
-----------------
the trait `From<&str>` is not implemented for `iced_native::element::Element<'_, (), iced_graphics::renderer::Renderer<iced_wgpu::backend::Backend>>`

**Comment by arjunair86** [comment]
Docs are often outdated. Here's the one which should work.

```
use iced::{Sandbox, Settings, Text};

pub struct App;

fn main() {
    App::run(Settings::default()).unwrap();
}

impl Sandbox for App {
    type Message = ();

    fn new() -> Self {
        Self
    }

    fn title(&self) -> String {
        "Title".to_string()
    }

    fn update(&mut self, _message: Self::Message) {
        todo!()
    }

    fn view(&mut self) -> iced::Element<'_, Self::Message> {
        Text::new("Hello World").into()
    }
}
```

**Reply by IwateKyle** [reply]
Thank you.  
1) Is there a better place to ask these questions
2) Do you think I should hold off trying to learn / use this tool until it's stable

**Reply by arjunair86** [reply]
1. I believe you asked the question in the right place.
2. You can checkout the [projects](https://github.com/iced-rs/iced/issues/355) built using iced and get some idea of its usage.

**Comment by hecrj** [comment, marked as answer]
The docs hosted in https://docs.iced.rs/iced are for the `master` branch. If you are using the latest release on crates.io, the proper documentation is hosted in https://docs.rs/iced, as with any other crate.

**Reply by IwateKyle** [reply]
Thank you.  It took me a minute to understand what you meant.  But, now I understand.  I went to the crates.io and looked up the documentation and the Hello World example was indeed different than the one I found online.  I guess I need to pay closer attention to the source of the documentation I'm reading.  Thank you again, that was helpful.
